haystack/document_store/faiss.py

```diff
@@ -88,15 +88,15 @@ def __init__(
             embedding_field=embedding_field, progress_bar=progress_bar
         )
 
-        if similarity == "dot_product":
+        if similarity == "dot_product" or similarity == 'cosine':
             self.similarity = similarity
             self.metric_type = faiss.METRIC_INNER_PRODUCT
         elif similarity == "l2":
             self.similarity = similarity
             self.metric_type = faiss.METRIC_L2
         else:
-            raise ValueError("The FAISS document store can currently only support dot_product similarity. "
-                             "Please set similarity=\"dot_product\"")
+            raise ValueError("The FAISS document store can currently only support dot_product, cosine and l2 similarity. "
+                             "Please set similarity to one of the above.")
```

Comment:
Could you also update the docstring to reflect this? (i.e. line 70)

Reply:
The doc string currently says:
```
The similarity function used to compare document vectors. 'dot_product' is the default sine it is
                   more performant with DPR embeddings. 'cosine' is recommended if you are using a Sentence BERT model.
```
This is still correct isn't it? Or would you like me to add L2 here? I'm not sure what the docstring should say.

Reply:
Ahhh sorry I made a mistake here. Nevermind